monai/apps/auto3dseg/auto_runner.py

```diff
@@ -0,0 +1,200 @@
+import os
+
+import torch
+
+from monai.apps.auto3dseg import DataAnalyzer, BundleGen, AlgoEnsembleBuilder, AlgoEnsembleBestN
+from monai.apps.utils import get_logger
+
+from monai.bundle import ConfigParser
+from monai.utils.enums import AlgoEnsembleKeys
+
+logger = get_logger(module_name=__name__)
+
+
+class AutoRunner:
+    """
+    Auto3Dseg interface for minimal usage
+
+    Args:
+        data_src_cfg_name: path to a configuration file (yaml) that contains datalist, dataroot, and other params.
+                The config will be in a form of {"modality": "ct", "datalist": "path_to_json_datalist", "dataroot":
+                "path_dir_data"}
+        work_dir: working directory to save the intermediate results
+        analyze: on/off switch for data analyzer
+        algo_gen: on/off switch for algoGen
+        train: on/off switch for sequential schedule of model training
+        no_cache: if no_cache is True, it will reset the status and not use any previous results
+
+    Examples:
+
+        ..code-block:: python
+            work_dir = "./work_dir"
+            filename = "path_to_data_cfg"
+            runner = AutoRunner(data_src_cfg_filename, work_dir)
+            runner.run()
+
+        ..code-block:: python
+            work_dir = "./work_dir"
+            filename = "path_to_data_cfg"
+            runner = AutoRunner(data_src_cfg_filename, work_dir)
+            train_param = {
+                "CUDA_VISIBLE_DEVICES": [0],
+                "num_iterations": 8,
+                "num_iterations_per_validation": 4,
+                "num_images_per_batch": 2,
+                "num_epochs": 2,
+            }
+            runner.set_training_params(train_param)  # 2 epochs
+            runner.run()
+
+    Notes:
+        Expected results in the work_dir as below.
+            .
+            ├── algorithm_templates
```

Comment:
I think we can make this example shorter with "..." in it.

Thanks.

Reply:
Made it shorter. Thanks!